routes-schema.ts

```diff
@@ -0,0 +1,53 @@
+import { JSONSchema7 } from "json-schema";
+
+export const schema: JSONSchema7 = {
```

Comment:
build this as json and make it available for developes to use it using `#yaml-schema-language-server`
Update readme about its usage